docs/courses/hints_for_inhomogeneous_channel_dist.rst

```diff
@@ -0,0 +1,113 @@
+.. _hints_for_inhomogeneous_channel_distribution:
+
+Overview of the Tasks
+=====================
+
+1.
+    Specify the
+
+        - subset
+        - distance metric ``p``
+        - parameter ``param`` that depends on distance
+        - function ``f`` that governs the relationship between the parameter and the distance metric
+
+2.
+    Verify the implementation.
+
+Details
+=======
+
+This involves a lot of steps--remember to save intermediate results to session files!
+
+**Act 1: Specify the subset**
+
+On the CellBuilder's Subsets page, create the subset.
+
+**Act 2: Specify the distance metric**
+
+Do this with a SubsetDomainIterator.
+
+- Create a SubsetDomainIterator.
+
+    - Click on the subset, then click on "Parameterized Domain Page"
+    - Click on "Create a SubsetDomainIterator".
+
+    The name of the SubsetDomainIterator will appear in the middle panel of the CellBuilder. It will be called name_x, where *name* is the name of the subset.
+
+- Use the SubsetDomainIterator to specify the distance metric.
+
+    - Click on name_x.
+
+    The right panel of the CellBuider will show the controls for specifying the distance metric. Default is path length in um from the root of the cell (0 end of soma).
```

Comment:
Since we're redoing this, we ought to say "path length in µm from the root"... i.e. replace um with µm.